calico-policies/private-network-isolation/br-sao/allow-private-services.yaml

```diff
@@ -55,6 +55,10 @@ spec:
       - 166.9.83.12/32
       - 166.9.82.12/32
       - 166.9.84.12/32
+      # New IP addresses added on 10/19/2023
```

Comment:
Looks like these are the ca-tor IPs?

Reply:
These three IPs:
```
      - 166.9.247.151/32
      - 166.9.247.182/32
      - 166.9.247.201/32
````

match the ones in `calico-policies/private-network-isolation/ca-tor/allow-private-services-pods.yaml` and also the ca-tor IPs in the documentation.  I don't think they are br-sao IPs, or if they are, the docs are wrong and the IPs added in the other br-sao file in this PR are wrong